builders/manage/developer-portal/.pages

```diff
@@ -8,4 +8,5 @@ nav:
   - 'Pause Transactions': 'pause-transactions.md'
   - 'Impersonate Accounts': 'impersonate.md'
   - 'Mint Tokens': 'minting.md'
+  - 'Smart EVM - Whitelist Smart Contracts Deployment': 'smart-contracts-creation-filter.md'
```

Comment:
```suggestion
  - 'Whitelist Smart Contracts Deployers': 'smart-contracts-creation-filter.md'
```

Reply:
After rendering the page it does not look that bad. @eshaben what do you think?

![image](https://github.com/moondance-labs/tanssi-docs/assets/64150856/92949575-311f-4224-b200-b061e606f890)